src/main/java/sg/ncl/MainController.java

```diff
@@ -1161,7 +1138,7 @@ public String editAccountDetails(
                         MyErrorResource error = objectMapper.readValue(responseBody, MyErrorResource.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "error" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)